src/App.tsx

```diff
@@ -2,12 +2,31 @@ import { Routes, Route } from 'react-router-dom'
 import { LoginPage } from '@/pages/LoginPage'
 import { NotFoundPage } from '@/pages/NotFoundPage'
 import { Home } from '@/pages/Home'
+import { MyPage } from '@/pages/MyPage'
+import { PrivateRoute } from '@/routes/PrivateRoute'
+import { OrderPage } from '@/pages/OrderPage/OrderPage'
 
 function App() {
   return (
     <Routes>
       <Route path="/" element={<Home />} />
       <Route path="/login" element={<LoginPage />} />
+      <Route
+        path="/my"
+        element={
+          <PrivateRoute>
+            <MyPage />
+          </PrivateRoute>
+        }
+      />
+      <Route
+        path="/order/:id"
+        element={
+          <PrivateRoute>
+            <OrderPage />
+          </PrivateRoute>
+        }
+      />
```

Comment:
```suggestion
      <PrivateRoute
        path="/my"
        element={
          <MyPage />
        }
      />
      <PrivateRoute
        path="/order/:id"
        element={
          <OrderPage />
        }
      />
```
이렇게 쓰이도록 하려면 어떤 방식으로 만들어져야 좋을까요!?

Reply:
PrivateRoute 컴포넌트에서 <Route>를 리턴하려고 했지만 인식하지 못했습니다. 찾아보니 React Router v6 이상에서 <Route>는 <Routes> 컴포넌트의 직접 자식이어야만 동작하는 것 같아 Outlet을 적용해 중첩 라우팅 구조로 수정했습니다.

Reply:
헛... 몰랐던 사실이네요 ㅎㅎ
덕분에 하나 알아갑니다.